pkg/util/ticker/metrics.go

```diff
@@ -1,33 +1,33 @@
-package metrics
+package ticker
 
 import (
 	"github.com/prometheus/client_golang/prometheus"
 	"github.com/prometheus/client_golang/prometheus/promauto"
 )
 
-type Ticker struct {
+type TickerMetrics struct {
```

Comment:
Probably Metrics would be enough because `ticker` is reflected in the package name

Reply:
Good call, `ticker.Metrics` is real clean!